pkg/util/string/string.go

```diff
@@ -0,0 +1,74 @@
+package string
```

Comment:
As a followup to the comment above, this module might be gone as currently it mixes up distinct structures: some methods are string array specific, some expect arrays to be sets, and there is no implication on order. I'd suggest keeping this logic inside rollout block specific middleware and not touching strings at all.